profiling/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "datadog-php-profiling"
 version = "0.96.0"
 edition = "2021"
 license = "Apache-2.0"
-rust-version = "1.64"
+rust-version = "1.71"
```

Comment:
While we currently say 1.71 in our docs, the code compiles perfectly fine with 1.70, which is the newest available via the debian package manager.

The differences between 1.70 and 1.71 are also really small...

Reply:
Oh yes, you are right, I reverted that change.